developer_manual/client_apis/WebDAV/index.rst

```diff
@@ -0,0 +1,242 @@
+
+----
+Creating folders (rfc4918_)
+----
+
+A folder can be created by sending a :code:`MKCOL` request to the folder.
+
+.. code::
+
+	MKCOL remote.php/dav/files/user/path/to/new/folder
+
+----
+Deleting files and folders (rfc4918_)
+----
+
+A file or folder can be created by sending a :code:`DELETE` request to the file or folder.
+
+.. code::
+
+	DELETE remote.php/dav/files/user/path/to/file
+
+When deleting a folder, it's contents will be deleted recursively.
+
+----
+Moving files and folders (rfc4918_)
+----
+
+A file or folder can be moved by sending a :code:`MOVE` request to the file or folder and specifying the destination in the :code:`Destination` header as full url.
+
+.. code::
+
+	MOVE remote.php/dav/files/user/path/to/file
+	Destination: https://cloud.example/remote.php/dav/files/user/new/location
+
+The overwrite behavior of the move can be controlled by setting the :code:`Overwrite` head to :code:`T` or :code:`F` to enable or disable overwriting respectively.
+
+----
+Copying files and folders (rfc4918_)
+----
+
+A file or folder can be copied by sending a :code:`COPY` request to the file or folder and specifying the destination in the :code:`Destination` header as full url.
+
+.. code::
+
+	COPY remote.php/dav/files/user/path/to/file
+	Destination: https://cloud.example/remote.php/dav/files/user/new/location
+
+The overwrite behavior of the copy can be controlled by setting the :code:`Overwrite` head to :code:`T` or :code:`F` to enable or disable overwriting respectively.
+
+----
+Settings favorites
+----
+
+A file or folder can be marked as favorite by sending a :code:`PROPPATCH` request to the file or folder and setting the :code:`oc-favorite` property
+
+.. code-block:: xml
+
+	PROPPATCH remote.php/dav/files/user/path/to/file
+	<?xml version="1.0"?>
+	<d:propertyupdate xmlns:d="DAV:" xmlns:oc="http://owncloud.org/ns">
+	  <d:set>
+		<d:prop>
+		  <oc:favorite>1</oc:favorite>
+		</d:prop>
+	  </d:set>
+	</d:propertyupdate>
+
+Setting the :code:`oc:favorite` property to 1 marks a file as favorite, setting it to 0 un-marks it as favorite.
+
+----
+Listing favorites
+----
+
+Favorites for a user can be retrieved by sending a :code:`REPORT` request and specifying :code:`oc:favorite` as a filter
+
+.. code-block:: xml
+
+	REPORT remote.php/dav/files/user/path/to/folder
+	<?xml version="1.0"?>
+	<oc:filter-files  xmlns:d="DAV:" xmlns:oc="http://owncloud.org/ns" xmlns:nc="http://nextcloud.org/ns">
+		 <oc:filter-rules>
+			 <oc:favorite>1</oc:favorite>
+		 </oc:filter-rules>
+	 </oc:filter-files>
+
+File properties can be requested by adding a :code:`<d:prop/>` element to the request listing the requested properties in the same way as it would be done for a :code:`PROPFIND` request.
+
+When listing favorites, the request will find all favorites in the folder recursively, all favorites for a user can be found by sending the request to :code:`remote.php/dav/files/user`
+
+----
+Settings tags
```

Comment:
Since these are not the "visible collaborative" tags, but user depending tags which have no UI anywhere, I wouldn't document them.